Marlin/temperature.cpp

```diff
@@ -787,10 +787,11 @@ void manage_heater() {
   // ELECTRO-PNEUMATIC REGULATOR CONTROL
   #if (ENABLED(E_REGULATOR) && ENABLED(PNEUMATICS))
   if (millis() - previous_millis_regulator_value > REGULATOR_CHECK_INTERVAL) {
-    
+    // Updates the regulator protection timers
+    Regulator__Update();
     previous_millis_regulator_value = millis();
     //Is output pressure more than what is available?
-    if((pressureRegulator() >= pressurePneumatic()) && (pressurePneumatic() >= 1)) {
+    if((pressureRegulator() >= pressurePneumatic()) && (pressurePneumatic() >= REGULATOR_LOW_P) && (pressureRegulator() != REGULATOR_NOT_PRESENT_VALUE)) {
```

Comment:
We should consolidate these two different checks we are doing, and handle them the same way.